Improbable/Stdlib/Improbable.Stdlib/FutureExtensions.cs

```diff
@@ -17,11 +17,13 @@ public static Task<T> ToTask<T>(this Future<T> future, CancellationToken cancell
                 {
                     cancellation.ThrowIfCancellationRequested();
 
-                    if (future.TryGet(out var value, 50))
+                    if (!future.TryGet(out var value, 50))
```

Comment:
Thinking about this, this should probably be a 0ms timeout on the future, then run an `await Task.Delay(TimeSpan.FromMilliseconds(50))` so we aren't blocking the thread this task is running on.